backend/src/index.ts

```diff
@@ -4,6 +4,7 @@ import mqttsServer from './mqttsServer';
 import 'dotenv/config';
 import websocketServer from './websocketServer/websocketServer';
 import httpServer from './httpServer';
+//~ process.env.NODE_TLS_REJECT_UNAUTHORIZED='0';
```

Comment:
Worth keeping it clean!

```suggestion
```

Reply:
yeah, indeed you're right, i have this habit of keeping stuff commented out for further referrence but we're in git so no harm if line disappears i guess 😅